devenv/cloud/eks-cdk/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eks-cdk",
+  "version": "0.1.0",
+  "bin": {
+    "eks-cdk": "bin/eks-cdk.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@types/jest": "^29.5.12",
+    "@types/node": "20.11.19",
+    "@types/source-map-support": "^0.5.10",
+    "aws-cdk": "2.133.0",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.1.2",
+    "ts-node": "^10.9.2",
+    "typescript": "~5.3.3"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-ecr": "^1.204.0",
+    "@aws-quickstart/eks-blueprints": "^1.14.1",
```

Comment:
Can you change these to use the latest aws cdk version? Looks like we're using AWS CDK v1 version of ecr and eks constructs which is deprecated.